backend/src/test/java/de/neuefische/capstone/backend/services/MyLoansServiceTest.java

```diff
@@ -72,7 +72,6 @@ void expectNewLoan_whenAddNewLoan() {
         when(myLoansRepository.findById(userId)).thenReturn(Optional.of(userDataToUpdate));
         when(myLoansRepository.save(userDataToUpdate)).thenReturn(userDataUpdated);
         LoanWithoutId actualLoan=myLoansService.addLoan(newLoanWithoutId);
-        //expectedLoan.setId(actualLoan.getId());
```

Comment:
another comment :)

Reply:
these are deleted ones 😅 i was looking for them for half an hour 😂